pkgs/development/python-modules/qds_sdk/default.nix

```diff
@@ -7,33 +7,41 @@
  mock,
  requests,
  six,
- urllib3 }:
+ urllib3,
+ fetchpatch }:
 
 buildPythonPackage rec {
   pname = "qds_sdk";
-  version = "1.12.0";
+  version = "1.15.2";
 
   # pypi does not contain tests, using github sources instead
   src = fetchFromGitHub {
     owner = "qubole";
     repo = "qds-sdk-py";
     rev = "V${version}";
-    sha256 = "18xhvlcfki8llv7fw2r5yfk20zds3gr78b4klwm9mkvhlhwds9rx";
+    sha256 = "0xxg9s0y6fz7vb1kab4q93q7ryi71z8x6q9qspm6s506yr3mc67l";
   };
 
-  propagatedBuildInputs = [ 
+  patch = [
+    (fetchpatch {
```

Comment:
Please add a comment as to why this patch is needed for this version.

Reply:
👌🏼